src/drivers/mb12xx/mb12xx.cpp

```diff
@@ -259,6 +277,38 @@ MB12XX::init()
 			debug("failed to create sensor_range_finder object. Did you start uOrb?");
 		}
 	}
+	usleep(200000);
```

Comment:
The usleep here holds off all other low-priority queues sensors as well, is it strictly necessary?